covalent_dispatcher/_db/dispatchdb.py

```diff
@@ -183,7 +183,6 @@ def __exit__(self, exc_type, exc_value, traceback):
 
     def _db_dev_path(self):
```

Comment:
Based on recent changes made by @FyzHsn .. this may no longer be needed... do check with him if possible

Reply:
got it, thanks @venkatBala , I removed this when fixing merge conflict